src/execution_plan/ops/shared/update_functions.c

```
@@ -249,7 +252,8 @@ void EvalEntityUpdates
 				break;
 			}
 
-			AttributeID attr_id = FindOrAddAttribute(gc, attribute, true);
+			AttributeID attr_id = property->attr_id;
+			ASSERT(attr_id != ATTRIBUTE_ID_NONE);
```

cppcheck

[error] Null pointer dereference

(nullPointer)